packages/clerk-js/src/ui/components/UserProfile/PasswordPage.tsx

```diff
@@ -21,7 +53,14 @@ export const PasswordPage = withCardStateProvider(() => {
     isRequired: true,
   });
 
-  const canSubmit = passwordField.value && passwordField.value === confirmField.value && passwordField.value.length > 7;
+  const sessionsField = useFormControl('signOutOfOtherSessions', '', {
+    type: 'checkbox',
+    label: localizationKeys('formFieldLabel__signOutOfOtherSessions'),
+  });
+
+  const isPasswordMatch =
+    passwordField.value && passwordField.value === confirmField.value && passwordField.value.length > 7;
```

Comment:
❓ What is this `7` here? Password needs more than 7 characters?

Reply:
@anagstef Yes, clerk requires at least 8 characters for a user's password.

Reply:
@panteliselef Can we extract this `7` somewhere? It's getting lost here. We can introduce an `isValidPassword` method and a static const like `PASSWORD_MIN_LENGTH`, or something similar.